src/pages/iou/IOUModal.js

```diff
@@ -29,11 +37,24 @@ const propTypes = {
         error: PropTypes.bool,
     }).isRequired,
 
+    // Personal details of all the users
+    personalDetails: PropTypes.objectOf({
+        // Primary login of participant
+        login: PropTypes.string,
+
+        // Display Name of participant
+        displayName: PropTypes.string,
 
+        // Avatar url of participant
+        avatar: PropTypes.string,
+    }).isRequired,
```

Comment:
Seeing an error here. I think it needs to use shape? 

```
Failed prop type: Property `personalDetails` of component `IOUModal` has invalid PropType notation inside objectOf.
```

Reply:
ah yes, needs to be `shape`